code/modules/mapping/station_datums.dm

```diff
@@ -23,6 +23,12 @@
 	webmap_url = "https://webmap.affectedarc07.co.uk/maps/paradise/cerestation/"
 	min_players_random = 60
 
+/datum/map/emeraldstation
+	fluff_name = "NSS Diagoras"
+	technical_name = "EmeraldStation"
+	map_path = "_maps/map_files/stations/emeraldstation.dmm"
+	webmap_url = "https://affectedarc07.blob.core.windows.net/mdb2/images/27286452/29268698805/a/_maps_map_files_stations_emeraldstation/0-added.png"
```

Comment:
```suggestion
	webmap_url = "https://affectedarc07.co.uk/emerald.html"
```

Reply:
should already be changed over, that was my bad

Reply:
Maybe you didn't commit it by accident?

Reply:
oops, i did not push it yeah